Завершаем вёрстку всех компонентов

AlexPolyakovWeb opened the pull request:
Модальное окно, пока без поп-апа

---
:mortar_board: [Завершаем вёрстку всех компонентов](https://up.htmlacademy.ru/htmlcss/33/user/1967959/tasks/12)

:boom: https://htmlacademy-htmlcss.github.io/1967959-sedona-33/10/

AlexPolyakovWeb committed:
Попапы, слайдеры, формы

Модальное окно, пока без поп-апа

AlexPolyakovWeb committed:
Докодил и поп-ап с тултипом

чтоб уж заодно все посмотреть

AlexPolyakovWeb committed:
Сверстал модальное вместе с тултипом

4err reviewed (changes requested):
Что-то у Дети ни инпут ни кнопки не кликабельны.

Ну и пора доделывать пиксель перфект уже, подгонять все расхождения.

AlexPolyakovWeb committed:
Главная по перфек тпикселю

AlexPolyakovWeb committed:
Вынес тул из попапа

Вынес тул, кнопки стали работать как я и предполагал а вот логика при наведении перестала срабатывать не появляется попап (((( выручай

AlexPolyakovWeb committed:
))) додумал всё работает

AlexPolyakovWeb committed:
Сверстал всё по пёрфеке

AlexPolyakovWeb committed:
Жирность в цифрах выбора людей

AlexPolyakovWeb committed:
Поправил согласно новым макетам

AlexPolyakovWeb committed:
Исправил замечания по макету

AlexPolyakovWeb committed:
Поправил отображение на меньших экранах

AlexPolyakovWeb committed:
Допилил пёрфект

с инпутами в фильтре в каталоге так и не понял ......извини, буду ждать помощи а то уже задница плоская ))

4err reviewed (changes requested):
![image](https://user-images.githubusercontent.com/3590683/158529033-418d34cc-6487-4116-9f28-7b62004a26ec.png)
Для ховеров и фокусов та же схема. Ты должен стилизовать не инпут, а спан, в котором у тебя отрисовывается дизайн инпута.
`.control-input[type="checkbox"]:focus/:hover + .control-mark::before`
И уже описываешь свои правила. 

Внезапно у поиска и лайка нет фокуса. Нужно добавить хреф ссылкам.

индекс и каталог не проходят валидацию.

Зеленую иконку у сердечка лучше заменить на стилизованный спан, т.к. число у нас постоянно будет меняться, вряд ли это каждый раз будут новые свг.

AlexPolyakovWeb committed:
Всё сделал кроме валидатора

AlexPolyakovWeb committed:
Валидатор

Индекс — прошёл/каталог — прошёл
CSS — ругается на дерективы от академии в остальном всё хорошо

AlexPolyakovWeb committed:
Сделано шеф )))
